spec/validate.py

```diff
@@ -32,9 +32,28 @@
         "file": os.path.join(_BASE_DIR, "view_schema.yaml"),
         "plural": "views",
     },
+    "analyzer": {
+        "file": os.path.join(_BASE_DIR, "analyzer_schema.yaml"),
+        "plural": "analyzers",
+    },
 }
 
 
+def get_schema_type_paths(schema_type, directory=None):
+    if schema_type not in _VALID_SCHEMA_TYPES.keys():
+        raise ValueError(f"No validation schema found for '{schema_type}'")
+    if directory is None:
+        type_dir_name = _VALID_SCHEMA_TYPES[schema_type]["plural"]
+        directory = _CONF["spec_paths"][type_dir_name]
+
+    paths = []
+    for path in glob.iglob(os.path.join(directory, "**", "*.*"), recursive=True):
```

Comment:
Hmm ... should I be using that consolidated JSON/YAML globbing fixed-up function here ...

Reply:
NVM DRYed if up by using this function everywhere else